packages/vue/src/runtime.ts

```diff
@@ -240,11 +254,34 @@ export const defineContainer = <Props, VModelType = string | number | boolean>(
       Container.props[componentProp] = DEFAULT_EMPTY_PROP;
     });
 
+    /**
+     * Add emit props to the component.
+     * This is necessary for Vue to know
+     * which events to listen to.
+     * @see https://v3.vuejs.org/guide/component-custom-events.html#event-names
+     */
+    emitProps.forEach((emitProp) => {
+      // @ts-expect-error
```

Comment:
Instead of expecting an error can we just define a new variable and cast it to the type we expect it to be?